### PR TITLE
Use less inflection on aliasing.

### PR DESCRIPTION
**Pull request by dereuromark**
Let's see if this resolves https://github.com/cakephp/cakephp/issues/13448

### CODE REVIEW

File: src/ORM/Locator/TableLocator.php [hunk 1]
```diff
@@ -157,7 +157,7 @@ public function getConfig(?string $alias = null): array
      * key in the registry. This means that if two plugins, or a plugin and app provide
      * the same alias, the registry will only store the first instance.
      *
-     * @param string $alias The alias name you want to get.
+     * @param string $alias The alias name you want to get. Must be in CamelCase format.
```
Comment (dereuromark) [inline src/ORM/Locator/TableLocator.php]: Or "Should be ..." maybe?

**Comment by ADmad** [comment]
No test failures :slightly_smiling_face:.

**Comment by ADmad** [comment]
Could you add a test showing what error/exception to expect (if at all) if some uses non conventional alias.

**Comment by dereuromark** [comment]
I dont expect errors, more like using different auto created objects, one with core fallback. 
If someone has time to add tests, I'd appreciate the help.

**Comment by ADmad** [comment]
You are right, there won't be any error.

**Review by ADmad — APPROVED** [review]

**Pull request merged**